File: pset1/cash/cash.c
#include<stdio.h>
#include<math.h>
#include<cs50.h>

int main(void)
{
    //taking input
    float dollars;
    do
    {
        dollars = get_float("Change owed: ");
    }
    while (dollars < 0.0001);

    //converting
    int cents = round(dollars * 100);
    int cnt = 0;

    //nuber of querters
    cnt += cents / 25;
    cents %= 25;

    //nuber of dimes
    cnt += cents / 10;
    cents %= 10;

    //nuber of nickels
    cnt += cents / 5;
    cents %= 5;

    //nuber of pennys
    cnt += cents / 1;
    cents %= 1;

    printf("%i\n", cnt);

    return 0;
}

File: pset6/cash/cash.py
from cs50 import get_float

while True:
    owed = get_float("Change owed: ")
    if owed >= 0:
        break

owed *= 100

coins = owed // 25
owed %= 25
coins += owed // 10
owed %= 10
coins += owed // 5
owed %= 5
coins += owed

print(int(coins))
